subscriptions/reporting/subscription_reports.rst

```diff
@@ -0,0 +1,124 @@
+  shifting our focus from a quarterly or yearly approach to a long-term one.
+
+Check out the different kinds of reports you can access from the **Odoo Subscriptions** application.
+
+Subscriptions analysis report
+=============================
+
+Go to :menuselection:`Subscriptions --> Reporting --> Subscriptions`. From there, you can change the
+*Measures*. By default, Odoo uses the *Monthly Recurring Revenue*. In addition to that, you can
+choose *Quantity*, *Recurring Price*, *Yearly Recurring Revenue* and *Count*. For this example,
+*Quantity* is added. This way, you can review both of these measures at the same time. You can even
+*Group By Start Date* and, more precisely, by *Week*, to get a clear view of your report.
+
+.. image:: media/subscriptions-analysis-report.png
+  :align: center
+  :alt: Subscriptions analysis report in Odoo Subscriptions
+
+Retention analysis report
+=========================
+
+Go to :menuselection:`Subscriptions --> Reporting --> Retention`. The default measure applied is
+*Count*, but you can change to the appropriate one for you. For the example below,
+*Monthly Recurring Revenue* was chosen, and the *Month* periodicity remains intact.
+By using these criteria, you can see the progression of the retention from its start.
+
+.. image:: media/retention-analysis-report.png
+  :align: center
+  :alt: Retention analysis report in Odoo Subscriptions
+
+Revenue KPIs report
+===================
+
+Go to :menuselection:`Subscriptions --> Reporting --> Revenue KPIs`. From there, you can check
+different KPIs: *Monthly Recurring Revenue*, *Net Revenue*, *Non-Recurring Revenue*,
+*Revenue per subscription*, *Annual Run-Rate*, *Lifetime Value*, and more. You can also filter this
+information on subscriptions, companies, and sales teams. This is useful if you are looking for
+specific information.
+
+.. image:: media/revenue-KPIs-report.png
+  :align: center
+  :alt: Revenue KPIs report in Odoo Subscriptions
+
+The example below shows the *Monthly Recurring Revenue* detailed report. At the moment, there is no
+data, which is the typical scenario for a new business. But, as your company grows over the months,
+these graphs get populated with more and more data. Once again, you can filter these specific KPIs
+on subscriptions, companies, and sales teams.
+
+.. image:: media/detailed-MRR-report.png
+  :align: center
+  :alt: Detailed MRR report in Odoo Subscriptions
+
+Salesperson dashboard report
+============================
+
+Go to :menuselection:`Subscriptions --> Reporting --> Salesperson Dashboard`. This page gives you a
+summary of your *Monthly Recurring Revenue*, your *Non-Recurring Revenue*, your
+*Subscription modifications*, and your *Non-Recurring Invoices* for each of your salespeople. You
+can choose the period you want to apply and the salesperson you want to analyze.
+
+.. image:: media/salesperson-dashboard-report.png
```

Comment:
It would be nice to see numbers here, but I have tried myself and it is not that simple to create data :( ^^

Reply:
Yep, I also tried but it appears as fake data so I preferred not to put anything.